src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -283,20 +283,44 @@
     "themes": [
         {
             "name": "light",
-            "window":{
+            "window":
+            {

```

Comment:
Can you please reformat this file in VS Code with default formatting settings before we merge this? By default braces are attached to the line with the colon.